### DISCUSSION TITLE
In the scenario of an actor cluster, link and monitor are not working properly

### DISCUSSION THREAD

**Discussion by phil-skillwon** [slawlor/ractor#172] [Q&A] [answered]
I created a Sampler actor on node A and a Processor actor on node B. Then, on node B, I accessed the Sampler on node A through the group, and used link to make the Sampler a supervisor. After sleeping for 1 second, I stopped the Processor on node B. However, the Sampler on node A did not receive any supervision events.

Then, I tested the monitor method in the same way, like this: Sampler.monitor(Processor). After sleeping for 1 second, I stopped the Processor on node B. But the Sampler on node A still did not receive any supervision events.

Does ractor currently support this mechanism of lifecycle association between actors across clusters?

**Comment by slawlor** [comment, marked as answer]
It does not support link/monitor across a network boundary today. It's something we hope to have working at some point in the future, however it's still a roadmap item for the time being.
